console/src/main/webapp/manager/app/assets/fonts

```diff
@@ -0,0 +1 @@
+../../fonts/
```

Comment:
Why still using this fonts dir, and not symlink directly to bootstrap's one in node_modules?